tests/device_tests/test_msg_signtx_replacement.py

```diff
@@ -100,33 +99,38 @@ def test_p2pkh_fee_bump(client):
         orig_index=1,
     )
 
+    tt = client.features.model == "T"
+    responses = [
+        resp
```

Comment:
instead of this comprehension:

```
client.set_expected_responses([
	...,
    request_output(0, TXHASH_beafc7),
    (tt, request_orig_input(0, TXHASH_50f6f1)),
    (tt, request_orig_output(0, TXHASH_50f6f1)),
    request_input(0),
	...
])
```

Reply:
568f503